plugin.xml

```diff
@@ -20,7 +20,7 @@
 
 <plugin xmlns="http://apache.org/cordova/ns/plugins/1.0"
            id="cordova-plugin-inappbrowser"
-      version="1.7.0-os">
+      version="3.0.0">
```

Comment:
I wonder if I should have changed the version to 3.0.0-OS.
Can you please confirm me that?

Reply:
Yes, that would indicate it has custom code added (or removed) by OutSystems. Please do so.